src/components/ble/HeartRateService.cpp

```diff
@@ -72,11 +73,11 @@ void HeartRateService::OnNewHeartRateValue(uint8_t heartRateValue) {
 }
 
 void HeartRateService::SubscribeNotification(uint16_t connectionHandle, uint16_t attributeHandle) {
-  if(attributeHandle == heartRateMeasurementHandle)
+  if (attributeHandle == heartRateMeasurementHandle)
```

Comment:
We should probably add braces to these at some point, but this can be done in a separate PR at some later point

Reply:
Once clang-format 15 is out, that can be done automatically with the `InsertBraces` option.